docs/source/onedal/appendix/decision-tree.rst

```diff
@@ -0,0 +1,60 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _dt:
+
+Decision Tree
+*************
+
+.. toctree::
+   :maxdepth: 1
+
+
+Decision trees partition the feature space into a set of hypercubes,
+and then fit a simple model in each hypercube. The simple model can
+be a prediction model, which ignores all predictors and predicts the
+majority (most frequent) class (or the mean of a dependent variable
+for regression), also known as 0-R or constant classifier.
+
+Decision tree induction forms a tree-like graph structure as shown in
+the figure below, where:
+
+-  Each internal (non-leaf) node denotes a test on features
+-  Each branch descending from node corresponds to an outcome of the
+   test
+-  Each external node (leaf) denotes the mentioned simple model
+
+.. image:: images/decision-tree-structure.png
+  :width: 600
```

Comment:
we need alt text for this figure with `:alt:` directive

Reply:
fixed

Reply:
I suggest this brief summary for the alt text:

```
:alt:  A binary tree with seven nodes: the root, two internal nodes, and four leaf nodes. Each non-leaf node denotes a test on one of the features.
```